tests/integration/apigateway/test_apigateway_api.py

```diff
@@ -0,0 +1,11 @@
+from localstack.utils.files import load_file
+from tests.integration.apigateway import OPENAPI_SPEC_PULUMI_JSON
+
+
+def test_import_rest_api(import_apigw, snapshot):
```

Comment:
Not too exhaustive for now because we will be moving "import api" tests from existing modules.